packages/net/http/serverconnection.pony

```diff
@@ -1,5 +1,5 @@
-use "net"
 use "collections"
+use "net/tcp"
 
 actor _ServerConnection
```

Comment:
I think this file should be named `_server_connection.pony`, because the principal type declared in the file is `_ServerConnection`.